loopr-ios/Setting/SettingManageWallet/ChangeWalletName/SettingChangeWalletNameViewController.swift

```diff
@@ -8,22 +8,69 @@
 
 import UIKit
 
-class SettingChangeWalletNameViewController: UIViewController {
+class SettingChangeWalletNameViewController: UIViewController, UITextFieldDelegate {
 
     var appWallet: AppWallet!
+    var nameTextField: UITextField = UITextField()
+    var nameFieldUnderLine: UIView = UIView()
+    var saveButton: UIButton = UIButton()
     
     override func viewDidLoad() {
         super.viewDidLoad()
 
+        // Setup UI in the scroll view
+        let screensize: CGRect = UIScreen.main.bounds
+        let screenWidth = screensize.width
+        // let screenHeight = screensize.height
+        
+        let originY: CGFloat = 30
+        let padding: CGFloat = 15
+        
         // Do any additional setup after loading the view.
         self.navigationItem.title = NSLocalizedString("Change Wallet Name", comment: "")
         setBackButton()
-
+        
+        nameTextField.delegate = self
+        nameTextField.tag = 0
+        nameTextField.inputView = UIView()
```

Comment:
Remove this line. This will disable the system keyboard.

Reply:
removed.